specs/pinecone_api.json

```diff
@@ -121,7 +121,7 @@
       "delete": {
         "summary": "Delete",
         "description": "The `Delete` operation deletes vectors, by id, from a single namespace.\nYou can delete items by their id, from a single namespace.",
-        "operationId": "delete",
+        "operationId": "delete1",
```

Comment:
Did you mean this to be delete1 or leftover from testing?

Reply:
Intentional - that's automatically generated (it's the old DELETE api, now superceded by the new POST api for /vectors/delete). We could override the operationId if we want, but I thought delete1 was reasonable as is.